content/20.treatments.md

```diff
@@ -105,6 +105,49 @@ Summarize the state of the antiviral approach.
 #### Broad-Spectrum Pharmaceuticals
 
 ##### Nutraceuticals
+Considering the current pandemic, scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies for which nutraceuticals hold some promise.
+Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value designed for the prophylaxis and treatment of disease [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].
+Nutraceuticals purported to boost the immune response, reduce  immunopathology, exhibit antiviral activities or prevent acute respiratory distress syndrome (ARDS) are being considered for their potential therapeutic value [@doi:ggkd3b].
+A host of potential candidates have been highlighted in the literature that target various aspects of the COVID-19 viral pathology, while others are thought to prime the host immune system.
+These include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@doi:10.1002/jmv.25707].
```

Comment:
```suggestion
These candidates include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@doi:10.1002/jmv.25707].
```
It's not totally clear to me if that is what you meant, so please fix if I'm misunderstanding!